src/components/CreateCourse/components/AuthorList/AuthorList.jsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from '../../../../common/Button';
+import { BUTTON_ADD_AUTHOR } from '../../../../constants';
+
+function AuthorList({ addAuthor, authorList }) {
+	const element = (
+		<ul>
+			{authorList.map((author) => {
+				return (
+					<li key={author.id} className='author'>
+						{author.name}{' '}
+						<Button
+							buttonText={BUTTON_ADD_AUTHOR}
+							onButtonClick={() => addAuthor(author.id)}
+						/>
+					</li>
+				);
+			})}
```

Comment:
You can remove "return" statement here.
Something like this `arr.map((it) => ( <li>{it}</li> ) )`